src/modules/onboarding.ts

```diff
@@ -94,6 +94,7 @@ export default class Onboarding {
     params: {
       starkKey: string,
       starkKeyYCoordinate: string,
+      referredByAffiliateLink?: string,
```

Comment:
nit: i dunno if this matters for documentation or something, but it'd be good to add this param in the comment above for consistencies sake